### PR TITLE
tts: 0.0.15.1 -> 0.1.2

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

https://github.com/coqui-ai/TTS/releases/tag/v0.1.0
https://github.com/coqui-ai/TTS/releases/tag/v0.1.1
https://github.com/coqui-ai/TTS/releases/tag/v0.1.2
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mweinelt** [comment]
python-crfsuite doesn't work yet.

```pytb
_________________ ERROR collecting tests/test_itemsequence.py __________________
ImportError while importing test module '/build/python-crfsuite-0.9.7/tests/test_itemsequence.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/2nhfiak8a30vw67mxksc2kdb69np2jcw-python3-3.8.9/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_itemsequence.py:5: in <module>
    import pycrfsuite
pycrfsuite/__init__.py:2: in <module>
    from ._pycrfsuite import *
E   ModuleNotFoundError: No module named 'pycrfsuite._pycrfsuite'
```

File: pkgs/development/python-modules/gruut-ipa/default.nix [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    owner = "rhasspy";
+    repo = pname;
+    rev = "df74f7dff562d868ad8088b7251f9e7206271d37";
+    sha256 = "0b2znvjngffxc0mlmqmcai8l22ff09kc39bj6f0lkjw735vclnzh";
+  };
+
+  postPatch = ''
+    patchShebangs bin/speak-ipa
+    substituteInPlace bin/speak-ipa \
+      --replace '${"\${src_dir}:"}' "$out/lib/${python.libPrefix}/site-packages:" \
```
Comment (dotlambda) [inline pkgs/development/python-modules/gruut-ipa/default.nix:23]: :rofl:

File: pkgs/development/python-modules/gruut-ipa/default.nix [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+buildPythonPackage rec {
+  pname = "gruut-ipa";
+  version = "0.9.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "rhasspy";
+    repo = pname;
+    rev = "df74f7dff562d868ad8088b7251f9e7206271d37";
```
Comment (dotlambda) [inline pkgs/development/python-modules/gruut-ipa/default.nix:16]: You should file an issue about missing tags.

Comment (mweinelt) [reply pkgs/development/python-modules/gruut-ipa/default.nix:16]: As soon as I get `python-crfsuite` running. If I don't then :man_shrugging:.

Comment (mweinelt) [reply pkgs/development/python-modules/gruut-ipa/default.nix:16]: https://github.com/rhasspy/gruut-ipa/issues/3

**Comment by mweinelt** [comment]
The gruut language packages don't contain the package data, because either they're missing or need to be generated, and I don't know how.

https://github.com/rhasspy/gruut/issues/10#issuecomment-874998082

**Comment by mweinelt** [comment]
https://shells.darmstadt.ccc.de/~hexa/0.1.2.wav

WFM.

File: pkgs/tools/audio/tts/default.nix [hunk 3]
```diff
@@ -12,25 +12,24 @@
...
 # If you upgrade from an old version you may have to delete old models from ~/.local/share/tts
 # Also note that your tts version might not support all available models so check:
-#   https://github.com/coqui-ai/TTS/releases/tag/v0.0.15.1
+#   https://github.com/coqui-ai/TTS/releases/tag/v0.1.2
 #
 # For now, for deployment check the systemd unit in the pull request:
 #   https://github.com/NixOS/nixpkgs/pull/103851#issue-521121136
 
 python3Packages.buildPythonApplication rec {
   pname = "tts";
```
Comment (Mic92) [inline pkgs/tools/audio/tts/default.nix:21]: I find the new test suite quite slow. Can we exclude some of the slow tests somehow?

Comment (mweinelt) [reply pkgs/tools/audio/tts/default.nix:21]: On my desktop the build including tests completes in under 10 minutes, which I personally think is bearable, to make sure we didn't miss anything.

Maybe it is time to create a module and disable the tests for endusers, just like we did for home assistant? Ultimately if the package isn't cached it is usually due to a tensorflow or pytorch issue, so I hardly think the tts build time matters alot.

Comment (Mic92) [reply pkgs/tools/audio/tts/default.nix:21]: Ok. It took 30 minutes on mine.

**Pull request merged**